server/sonar-server/src/main/java/org/sonar/server/es/StickyFacetBuilder.java

```diff
@@ -79,17 +78,17 @@ public AggregationBuilder buildStickyFacet(String fieldName, String facetName, i
       .subAggregation(facetTopAggregation);
   }
 
-  public BoolFilterBuilder getStickyFacetFilter(String... fieldNames) {
-    BoolFilterBuilder facetFilter = FilterBuilders.boolFilter().must(FilterBuilders.queryFilter(query));
-    for (Map.Entry<String, FilterBuilder> filter : filters.entrySet()) {
+  public BoolQueryBuilder getStickyFacetFilter(String... fieldNames) {
+    BoolQueryBuilder facetFilter = QueryBuilders.boolQuery().must(QueryBuilders.queryFilter(query));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "queryFilter"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)